config.js

```diff
@@ -153,6 +153,10 @@ Object.keys(clockJobConfig.intervals).forEach((jobName) => {
   const defaultValue = clockJobConfig.intervals[jobName];
   const envValue = pe[`CLOCK_JOB_INTERVAL:${jobName}`];
   const value = envValue || defaultValue;
+  console.log('Configuring clock job intervals: ' +
```

Comment:
Any pros/cons of using: 

console.log(`This is a very long single line string which might be used \
to display assertion messages or some text. It has much more than \
80 symbols so it would take more than one screen in your text \
editor to view it. ${someVar}`);

Reply:
This is just a temporary message to help debug in a heroku environment :)